lib/ansible/plugins/action/uri.py

```diff
@@ -7,6 +7,8 @@
 
 import os
 
+from copy import deepcopy
```

Comment:
Sorry, I forgot to mention about changelog for this change. 
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Added